src/components/CaptureTable.js

```diff
@@ -69,26 +76,29 @@ const columns = [
     label: 'Planter ID',
   },
   {
-    attr: 'payment',
-    label: 'Payment',
+    attr: 'deviceIdentifier',
+    label: 'Device Identifier',
     noSort: true,
-    renderer: () => 'pending',
   },
   {
-    attr: 'country',
-    label: 'Country',
+    attr: 'planterIdentifier',
+    label: 'Planter Identifier',
     noSort: true,
-    renderer: () => 'pending',
+  },
+  {
+    attr: 'verificationStatus',
+    label: 'Verification Status',
   },
```

Comment:
Set this to `noSort: true,`
There's no `verificationStatus` attribute in the capture object, so it's not directly sortable using the LoopBack query.